src/index.html

```diff
@@ -6,17 +6,327 @@
+              tracking so you know where your bike is and can track it anytime.
+            </p>
+          </article>
+
+          <article>
+            <div class="bike-details">
+              <img
+                src="./assets/details-3.png"
+                alt="Suporte para celular"
+                class="details-image-md-3"
+              />
+              <img
+                src="./assets/details-4.png"
+                alt="Detalhe do freio da bicicleta"
+                class="details-image-sm-4"
+              />
+            </div>
+
+            <h3>Range & Integrated lights</h3>
+
+            <p class="description-details">
+              The removable battery has up to 70km battery autonomy and weighs
+              only 2.4 kg. Lights integrated into the frame give you always-on
+              visibility day and night.
+            </p>
+          </article>
+
+          <article>
+            <div class="bike-details">
+              <img
+                src="./assets/details-5.png"
+                alt="Suporte para celular"
+                class="details-image-sm-5"
+              />
+              <img
+                src="./assets/details-6.png"
+                alt="Detalhe do freio da bicicleta"
+                class="details-image-md-6"
+              />
+            </div>
+
+            <h3>Hydraulic disc brakes & Lightweight</h3>
+
+            <p class="description-details">
+              Brakes with total stopping power the second you make contact. The
+              removable battery has up to 70km battery autonomy and weighs only
+              2.4 kg. Lights integrated into the frame give you always-on
+              visibility day and night.
+            </p>
+          </article>
+        </section>
+
+        <div class="explore-button">
+          <button type="button">Explore</button>
+        </div>
+
+        <p
+          class="contact-title"
+          id="contacts"
+        >
+          Contact us
+        </p>
+
+        <div class="wrapper-contact">
+          <form onsubmit="this.reset(); return false;">
+            <fieldset>
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                required
+              />
+
+              <input
+                type="email"
+                name="email"
+                placeholder="Email"
+                required
+              />
+
+              <textarea
+                name="textarea"
+                placeholder="Message"
+                required
+              ></textarea>
+
+              <button type="submit">Send</button>
+            </fieldset>
+          </form>
+
+          <div class="contact-info">
+            <p>Phone</p>
+            <h4>+1 234 5555-55-55</h4>
+
+            <p>Email</p>
+            <h4>hello@miami.com</h4>
+
+            <p>Address</p>
+            <span class="contact-address">400 first ave.</span>
```

Comment:
`<a href="http://maps.google.com/?q=1200 Pennsylvania Ave SE, Washington, District of Columbia, 20003">...</a>`

https://stackoverflow.com/questions/27419509/open-google-map-with-specific-address-in-a-browser

Adapt your code to make this open in google maps